lbrynet/dht/blob_announcer.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, loop: asyncio.BaseEventLoop, node: 'Node', storage: 'SQLiteSt
         while batch_size:
             if not self.node.joined.is_set():
                 await self.node.joined.wait()
+            await asyncio.sleep(60, loop=self.loop)
+            if not self.node.protocol.routing_table.get_peers():
+                log.warning("No peers in DHT, announce round skipped")
+                continue
```

Comment:
should the delay be before the `continue`?

Reply:
It's before. `continue` will make it hit the delay